Collections URL Error "URL Error: No file found at https://raw.githubusercontent.com/Kometa-Team/Translations/master/defaults/en.yml"

### Version Number

2.0.2

### What branch are you on?

master

### Describe the Bug

Seeing this error all my collections.

URL Error: No file found at https://raw.githubusercontent.com/Kometa-Team/Translations/master/defaults/en.yml

Below is example from IMDB Top 250

> [2024-08-19 05:07:43,701] [meta.py:595]               [DEBUG]    | {'imdb_chart': 'top_movies', 'run_definition': [False, False], 'translation_key': 'imdb_top', 'ignore_blank_results': True, 'sort_title': '!03_<<title>>', 'url_poster': 'https://raw.githubusercontent.com/Kometa-Team/Default-Images/master/chart/IMDb%20Top%20250.jpg', 'collection_order': 'custom', 'sync_mode': 'sync', 'cache_builders': 1} |
[2024-08-19 05:07:43,701] [meta.py:596]               [DEBUG]    |                                                                                                    |
[2024-08-19 05:07:43,702] [builder.py:243]            [INFO]     |================================ Validating IMDb Top 250 Attributes ================================|
[2024-08-19 05:07:43,709] [kometa.py:920]             [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "//kometa.py", line 784, in run_collection                                                  |
                                                                 |     builder = CollectionBuilder(config, metadata, mapping_name, collection_attrs, library=library, extra=output_str) |
                                                                 |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ |
                                                                 |   File "/modules/builder.py", line 305, in __init__                                                |
                                                                 |     english = self.config.GitHub.translation_yaml("en")                                            |
                                                                 |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                            |
                                                                 |   File "/modules/github.py", line 93, in translation_yaml                                          |
                                                                 |     yaml = self._requests(f"{self.translation_url}{translation_key}.yml", yaml=True).data          |
                                                                 |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^               |
                                                                 |   File "/modules/github.py", line 29, in _requests                                                 |
                                                                 |     return self.requests.get_yaml(url, headers=self.headers, params=params)                        |
                                                                 |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                        |
                                                                 |   File "/modules/request.py", line 108, in get_yaml                                                |
                                                                 |     raise Failed(f"URL Error: No file found at {url}")                                             |
                                                                 | modules.util.Failed: URL Error: No file found at https://raw.githubusercontent.com/Kometa-Team/Translations/master/defaults/en.yml |
                                                                 |                                                                                                    |
[2024-08-19 05:07:43,711] [kometa.py:921]             [ERROR]    | URL Error: No file found at https://raw.githubusercontent.com/Kometa-Team/Translations/master/defaults/en.yml |
[2024-08-19 05:07:43,711] [kometa.py:932]             [INFO]     |                                                                                                    |

### Relevant Collection/Overlay/Playlist Definition

```yml
#Here is Movies section of config.xml

libraries:                           # This is called out once within the config.yml file
  Movies:                            # These are names of libraries in your Plex
    remove_overlays: false           # Set this to true to remove all overlays
    collection_files:
    - file: config/Movies.yml
    # - file: config/People.yml
    - default: seasonal                                 # Christmas, Halloween, etc.
      template_variables:                           # Canadian Thankgsgiving is a different date range. Otherwise, I want to ALWAYS see the seasonal
        sync_mode: append
        use_years: false
        use_patrick: false
        use_mother: false
        use_memorial: false
        use_father: false
        use_independence: false
        use_labor: false
        use_veteran: false
        use_aapi: false
        use_disabilities: false
        use_black_history: false
        use_lgbtq: false
        use_latinx: false
        use_women: false
        collection_section: '01'
        visible_shared: true
        visible_home: true
        collection_order: release.desc
        sort_by: release.desc
        sort_by_thanksgiving: audience_rating.desc
        schedule_valentine: range(02/01-02/15)
        schedule_easter: range(03/15-04/07)
        schedule_thanksgiving: range(11/14-11/28)
        schedule_christmas: range(11/20-12/31)
        trakt_list_christmas:
        - https://trakt.tv/users/hdlists/lists/christmas-movies
        - https://trakt.tv/users/del74/lists/christmas-movies
        imdb_list_halloween:
        - https://www.imdb.com/list/ls023118929/
        - https://www.imdb.com/list/ls000099714/
        - https://www.imdb.com/list/ls000058693/
        - https://www.imdb.com/list/ls000091321/
        - https://www.imdb.com/search/title/?genres=horror&keywords=haunted-house&sort=moviemeter,asc
        trakt_list_halloween:
        - https://trakt.tv/users/kairbear08/lists/halloween
        - https://trakt.tv/users/mybicycle/lists/halloween
        - https://trakt.tv/users/jayinftl/lists/halloween
        - https://trakt.tv/users/roswellgeek/lists/halloween
        tmdb_collection_halloween:
        - 91361     # Halloween Collection
        - 8581      # A Nightmare on Elm Street Collection
        - 1733      # The Mummy Collection
        - 8091      # Alien Collection
        - 9735      # Friday the 13th Collection
        - 2602      # Scream Collection
        trakt_list_valentine:
        - https://trakt.tv/lists/24637577
    - default: basic                 # This is a file within Kometa's defaults folder
      template_variables:
        use_lowest: false
        use_top: false
        collection_section: '02'
    - default: imdb                  # This is a file within Kometa's defaults folder
      template_variables:
        use_lowest: false
        use_top: false
        collection_section: '03'
    - default: trakt
      template_variables:
        use_anticipated: false
        use_collected: false
        use_popular: false
        use_recommended: false
        use_watched: false
        limit: 20
        collection_section: '04'
    - default: oscars
      template_variables:
        schedule: monthly(1)
        radarr_add_missing: true
        use_year_collections: false
        use_best_director: false
        collection_section: '05'
        data:
          starting: latest-60
          ending: latest
    - default: based
      schedule: monthly(1)
      template_variables:
        collection_section: '06'
    - default: universe
      schedule: monthly(1)
      template_variables:
        collection_order: release
        collection_section: '07'
        exclude:
        - mummy
        - askew
        append_data:
          karate: Karate Kid Universe
          bond: 007 Universe
          ipman: IP Man Universe
          halone: Home Alone Universe
          jpark: Jurassic Universe
          batman: Batman Universe
          spidy: SpiderMan Universe
          sups: Superman Universe
        trakt_list_karate: https://trakt.tv/lists/178
        trakt_list_bond: https://trakt.tv/lists/41
        trakt_list_ipman: https://trakt.tv/lists/724
        trakt_list_halone: https://trakt.tv/lists/101
        trakt_list_jpark: https://trakt.tv/lists/49
        trakt_list_batman: https://trakt.tv/lists/24112418
        trakt_list_spidy: https://trakt.tv/lists/11865292
        trakt_list_sups: https://trakt.tv/lists/3040887
        url_poster_karate: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/Karate%20Kid.jpg?raw=true
        url_poster_bond: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/James%20Bond%20007.jpg?raw=true
        url_poster_ipman: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/IP%20Man.jpg?raw=true
        url_poster_halone: https://theposterdb.com/api/assets/110662/view
        url_poster_jpark: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/Jurassic%20Park.jpg?raw=true
        url_poster_batman: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/Batman.jpg?raw=true
        url_poster_spidy: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/Spider-Man.jpg?raw=true
        url_poster_sups: https://github.com/meisnate12/Plex-Meta-Manager-Images/blob/master/franchise/Superman.jpg?raw=true
    - default: genre
      schedule: monthly(1)
      template_variables:
        collection_section: '08'
        exclude:
        - Christmas
        - Halloween
    - default: decade
      template_variables:
        schedule: monthly(1)
        limit: 50
        collection_section: '09'
    - default: actor
      template_variables:
#        schedule: monthly(1)
        collection_section: '10'
        style: rainier
        data:
          # limit: 10
          depth: 5
        # include:
        # - Elisha Cuthbert
        # - Yvonne Strahovski
        # - Jessica Alba
        # - Scarlett Johansson
        # - Emma Stone
        # - Mila Kunis
        # - Angelina Jolie
        # - Sydney Sweeney
        # - Adam Sandler
        # - Al Pacino
        # - Angelina Jolie
        # - Anne Hathaway
        # - Arnold Schwarzenegger
        # - Bruce Willis
        # - Chris Pratt
        # - Denzel Washington
        # - Dwayne Johnson
        # - Elizabeth Hurley
        # - Emma Watson
        # - George Clooney
        # - Harrison Ford
        # - Hugh Jackman
        # - Jason Statham
        # - Jennifer Connelly
        # - Jennifer Lawrence
        # - Jim Carrey
        # - Leonardo DiCaprio
        # - Matt Damon
        # - Mila Kunis
        # - Morgan Freeman
        # - Reese Witherspoon
        # - Robert De Niro
        # - Samuel L. Jackson
        # - Tom Cruise
        # - Tom Hanks
        # - Will Ferrell
    operations:
      # - schedule: weekly (friday)
        # mass_content_rating_update: mdb_commonsense0          # Update audience ratings with tmdb
        # mass_poster_update: tmdb                  # Updates every item’s poster to the chosen sites poster.
      # - schedule: weekly (saturday)
        # mass_critic_rating_update: tmdb
      # - schedule: weekly (sunday)
        # mass_user_rating_update: mdb_metacriticuser
    - schedule: weekly (monday)
      mass_audience_rating_update: imdb
    schedule_overlays: daily
    overlay_files:
    - remove_overlays: false                           # Set to true if you want to remove overlays
    - reapply_overlay: false                           # If you are doing a lot of testing and changes like me, keep this to true to always reapply overlays
#   - reset_overlays: tmdb                            # if you want to reset the poster to default poster from tmdb
    - default: ribbon                # This is a file within Kometa's defaults folder
      template_variables:
        use_all: false
        use_oscars: true
        use_oscars_director: true
        use_golden: true
        use_golden_director: true
    - default: resolution
      template_variables:
        use_1080p: false
        use_720p: false
        use_576p: false
        use_480p: false
    - default: audio_codec
      template_variables:
        horizontal_align: right
        use_flac: false
        use_pcm: false
        use_hra: false
        use_plus: false
        use_dtses: false
        use_dts: false
        use_digital: false
        use_aac: false
        use_mp3: false
        use_opus: false
    - default: ratings
      template_variables:
        rating1: audience                                                  # this is audience and audience: imdb
        rating1_image: imdb                                                # this is audience and mass_audience_rating_update: imdb
        rating1_font: config/overlays/fonts/Adlib.ttf                      # local font accessible to PMM
        rating1_font_size: 63                                              # adjusted font size to fit rating
        rating_alignment: horizontal
        horizontal_position: center                                         # the set of ratings is on the right of the poster
        vertical_position: top                                             # the set of ratings is on the top of the poster
    - default: commonsense
      template_variables:
        horizontal_align: left
        vertical_align: bottom
        vertical_offset: 25
        back_width: 220
```


### Logs

_No response_

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -34,4 +34,5 @@ Fixed #2195 an image on the docs was a dead link
 Fixes sort order of resolution collections
 Fixes #2228 ".any" not accepted for a variety of imdb_search parameters
 Fixes `streaming` defaults adding and removing items randomly
+Adds error information to help with #2201
 Various other Minor Fixes

--- a/modules/request.py
+++ b/modules/request.py
@@ -107,8 +107,14 @@ class Requests:
 
     def get_yaml(self, url, headers=None, params=None, check_empty=False):
         response = self.get(url, headers=headers, params=params)
-        if response.status_code >= 400:
+        if response.status_code == 401:
+            raise Failed(f"URL Error: Unauthorized - {url}")
+        if response.status_code == 404:
             raise Failed(f"URL Error: No file found at {url}")
+        if response.status_code == 429:
+            raise Failed(f"URL Error: Too many requests -  {url}")
+        if response.status_code >= 400:
+            raise Failed(f"URL Error: {response.status_code} on {url}")
         return YAML(input_data=response.content, check_empty=check_empty)
 
     def get_image(self, url, session=None):

